src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UploadContestedCaseDocumentController.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.controllers;
+
+import com.google.common.collect.ImmutableList;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.UploadContestedCaseDocumentsService;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.AUTHORIZATION_HEADER;
+
+@Slf4j
+@RestController
+@RequestMapping(value = "/case-orchestration")
+public class UploadContestedCaseDocumentController implements BaseController {
+
+    @Autowired
+    UploadContestedCaseDocumentsService service;
```

Comment:
this will work but for consistency make it `private final`, get rid of `Autowired` and add `RequiredArgsConstructor` ?

Reply:
Updated! Good spot